azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/NetworkInterface.java

```diff
@@ -39,36 +39,36 @@
     boolean isPrimary();
 
     /**
-     * @return the MAC Address of the network interface
+     * @return the Mac Address of the network interface
```

Comment:
Logged https://github.com/Azure/azure-sdk-for-java/issues/903 to track the general issue with capitalization, so that we can process this PR faster